lib/utils/responseHandler.js

```diff
@@ -1,33 +1,28 @@
 var Promise = require("bluebird");
 
-module.exports = (response, body) => {
+module.exports = (response) => {
   var outResponse = {
-    statusCode: response.statusCode,
+    statusCode: response.status,
     headers: response.headers,
-    body: body
+    body: response.data
   };
 
-  if (response.statusCode != 200) {
+  if (response.status != 200) {
     var errorResponse = outResponse;
     if (/\bapplication\/json\b/.test(response.headers['content-type'])) {
-      var responseBody = JSON.parse(body);
-      errorResponse.errorCode = responseBody.errorCode;
-      errorResponse.message = responseBody.message;
-      errorResponse.refId = responseBody.refId;
+      errorResponse.errorCode = response.data.errorCode;
+      errorResponse.message = response.data.message;
+      errorResponse.refId = response.data.refId;
       
-      if (responseBody.detail !== undefined) {
-        errorResponse.detail = responseBody.detail;
+      if (response.data.detail !== undefined) {
+        errorResponse.detail = response.data.detail;
       }
     } else {
-      errorResponse.message = body;
+      errorResponse.message = response.data;
     }
 
     return new Promise.reject(errorResponse);
-  } else if (response.headers['content-type'] === 'application/json;charset=UTF-8') {
-    outResponse.content = JSON.parse(body);
-    return outResponse;
-  } else {
```

Comment:
Does axios already handle `JSON.parse` in this case? Looks like it does from the test case you modified, but I just want to double check.

Reply:
Yes as far as I can tell `Axios` is handling this layer for us so we can remove these conditionals.